Mathlib/MeasureTheory/Measure/SumRestrict.lean

```diff
@@ -0,0 +1,82 @@
+/-
+Copyright (c) 2025 James Sundstrom. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: James Sundstrom
+-/
+import Mathlib.MeasureTheory.Measure.Restrict
+import Mathlib.Data.Set.Card
+
+/-!
+# Sum of restrictions of measures
+
+In this file we prove an upper bound on a sum of restrictions of a measure `μ`. This can be used
+to compare `∫ x ∈ X, f x ∂μ` with `∑ i, ∫ x ∈ (s i), f x ∂μ`, where `s` is a cover of `X`.
+-/
+
+open Set Finset
+
+lemma MeasureTheory.Measure.sum_restrict_le {α : Type*} [MeasurableSpace α] {ι : Type*}
```

Comment:
The `MeasurableSpace` argument has to match the one on which the measure is defined, so it's better to use an implicit argument.
```suggestion
lemma MeasureTheory.Measure.sum_restrict_le {α ι : Type*} {_ : MeasurableSpace α}
```